_pytest/fixtures.py

```diff
@@ -24,6 +24,10 @@
 from _pytest.outcomes import fail, TEST_OUTCOME
 
 
+class PseudoFixtureDef(object):
```

Comment:
im sorry for not being precise enough, by frozen attes class i meant http://www.attrs.org/en/stable/api.html#influencing-initialization - the `frozen=True` example

Reply:
Ah, ok. Will do! Thanks for the clarification!